Source/Turbo Navigator/TurboNavigator.swift

```diff
@@ -42,11 +45,15 @@ public class TurboNavigator {
     /// Convenience function to routing a proposal directly.
     ///
     /// - Parameter url: the URL to visit
+    /// - Parameter options: passed options will override any from the path configuration
     /// - Parameter parameters: provide context relevant to `url`
-    public func route(_ url: URL, parameters: [String: Any]? = nil) {
-        let options = VisitOptions(action: .advance, response: nil)
+    public func route(_ url: URL, options: VisitOptions? = nil, parameters: [String: Any]? = nil) {
```

Comment:
```suggestion
    public func route(_ url: URL, options: VisitOptions? = VisitOptions(action: .advance, response: nil), parameters: [String: Any]? = nil) {
```

I think this provides the same behavior but makes it even more clear what's going on in the public API.

Reply:
I like this. Much cleaner. I've removed `response: nil` since it's unused. Thanks!